pavement.py

```diff
@@ -65,11 +65,9 @@
 import shutil
 import subprocess
 import re
-try:
-    from hashlib import md5
-    from hashlib import sha256
-except ImportError:
-    from md5 import md5
+from hashlib import md5
+from hashlib import sha256
+
```

Comment:
Extra newline. Also, is something further down checking to see if sha256 was imported here?

Reply:
Nope.